src/widgets/principles/ui/principle-card/principle-card.tsx

```diff
@@ -10,9 +10,13 @@ export interface PrincipleCardProps {
 
 export const PrincipleCard = ({ title, description, icon }: PrincipleCardProps) => (
   <div className="principle-card">
-    <div className="accent" />
-    <span>{icon}</span>
-    <div className="card-title">{title}</div>
+    <div className="card-header">
+      <div className="icon-wrapper">
+        <div className="accent" />
+        <span>{icon}</span>
+      </div>
+      <div className="card-title">{title}</div>
+    </div>
     <div className="card-description">{description}</div>
     <div className="accent-corner" />
```

Comment:
the `accent-corner` and `accent` could be turned into `:before` pseudo elements in order to get rid of extra `div`s

Reply:
This is my variant in the `event-card` (maybe name `card-corner-accent` should be changed to `event-card-corner-accent` or make mixin more flexible
![image](https://github.com/user-attachments/assets/4f1ac24e-9e08-42c0-936f-bdbd2f87046c)